db-connector/src/main/java/uk/nhs/adaptors/connector/dao/PatientMigrationRequestDao.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.adaptors.connector.dao;
+
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+public interface PatientMigrationRequestDao {
+    @SqlUpdate("INSERT INTO public.patient_migration_request(id) VALUES (:id)")
```

Comment:
Perhaps we could remove this one? I don't want anyone to get confused and think we're going to set IDs manually. Those should be set using sequence.